app/templates/components/donations/donation-progress.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="information">
+  <div class="amount">
+    <div class="value">{{format-currency amountDonated trimZero=true}}</div>
+    <div class="descriptive-text">given each month</div>
+  </div>
+  <div class="percentage">
+    <div class="value">{{format-percentage percentage trimZero=true}}</div>
+    <div class="descriptive-text">of goal</div>
+  </div>
+</div>
+{{progress-bar-container percentage=percentage}}
+<div class="description">{{donationGoal.description}}</div>
```

Comment:
We could make the value an `<h{1,2,3}>` tag and the description a `<p>` tag. I personally prefer classes, though. Our HTML is not that complex that using classes would slow selection down, and the advantage is that we need to update tests less.

If we make the value an `<h2>` now, but decide that it needs to be an `<h3>` or even a `<div>` later, if that element keeps the class, the test will till pass. If we rely on a tag, though, the test will fail and have to be updated, even though the app itself still works.

Additionally, it's not really a header, so it doesn't make much sense to use an h tag just for the sake of not using a class. Really, I can't think of a tag closer fitting than a div. It's a text, displayed on it's own line, styled in a certain way. It's not a paragraph, or a header. We can make it, maybe a `<strong>`, but then we additionally need to make it display as block. If anything, it would be more appropriate to make it a `<strong>` wrapped in a `<div>`, but that would just add to the complexity.